shared/core/types/proto/actor.proto

```diff
@@ -20,16 +20,17 @@ enum StakeStatus {
 }
 
 // TODO(#555): Investigate ways of having actor specific params that are not part of the shared struct.
+//             Potentially having a separate struct for each actor type and a shared base.
 message Actor {
   ActorType actor_type = 1;
   string address = 2;
   string public_key = 3;
-  repeated string chains = 4; // Not applicable to `Validator` actors
+  repeated string chains = 4; // NB: Not applicable `Validator` actors
```

Comment:
nit:  :eyes: :smiling_face_with_tear: - should we add this to the makefile?

Reply:
We're going to have actor specific protos in the future.

What did you want to add to the Makefile?

Reply:
Apologies for the ambiguity :sweat_smile:, I was referring to `NB` as a `TODO` type.

Reply:
To me, NB is a `NOTE` (not a TODO). We have a TODO on top of the protocol actor proto:

![Screenshot 2023-04-24 at 10 49 47 AM](https://user-images.githubusercontent.com/1892194/234076423-9978175a-49eb-4d40-9cc0-8270e185b763.png)